From b11b5a22314e08964cc684715bbd6e0855bd71d4 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 11 Jul 2022 07:42:42 +0300
Subject: [PATCH] [SW-94367] gaudi3: remove todo comment

This patch removes TODO comment in pcie routine specific to
gaudi3.

Change-Id: Ibd18503879ae026ddfc0cfffb09761306d850f38
---

diff --git a/common/pcie.c b/common/pcie.c
index 6748c8b..d20ec52 100644
--- a/common/pcie.c
+++ b/common/pcie.c
@@ -143,7 +143,6 @@
 		enable_flr_handling_fsm();
 
 		/* disable HOT reset handling by HW state machine */
-		/* TODO: Check whether required for Gaudi3 */
 		regmap_writel(CPU_HOT_RST_CTRL_MASK,
 			      mmCPU_HOT_RST);
 
